openapi-diff/src/core/OpenApiDiff/Program.cs

```diff
@@ -43,5 +44,26 @@ private static int Main(string[] args)
 
             return 0;
         }
+
+
+        private static string GetOpenApiSpec(Uri uri)
+        {
+            if (uri.Scheme == Uri.UriSchemeHttp || uri.Scheme == Uri.UriSchemeHttps)
+            {
+                using (var client = new HttpClient())
+                {
+                    var response = client.GetAsync(uri).Result;
+                    return response.Content.ReadAsStringAsync().Result;
+                }
+            }
+
+            if (uri.Scheme == Uri.UriSchemeFile)
+            {
+                return File.ReadAllText(uri.AbsolutePath);
```

Comment:
can you please provide a Gist where this has been run successfully so we make sure there is no regression